napari/layers/points/_tests/test_points.py

```diff
@@ -1050,6 +1050,20 @@ def test_thumbnail():
     layer._update_thumbnail()
     assert layer.thumbnail.shape == layer._thumbnail_shape
 
+    # test with n_points > _max_max_points_thumbnail in 2D
```

Comment:
@kevinyamauchi can you put these into different tests? ie comment -> new function name, possibly with docstring. It would be cool if the docstring pointed to the GitHub issue it tests!

Reply:
No problem. Done!